packages/sdk-client-v3/tests/http.test/http-middleware.test.ts

```diff
@@ -52,23 +53,6 @@ describe('Http Middleware.', () => {
     ).toThrow()
   })
 
-  test('should throw if timeout is provided and `AbortController` is not.', () => {
```

Comment:
Also, any reason for removing this test case?

Reply:
Not really, this can actually happen (in older node versions), this check was done for older nodejs versions that doesn't have the AbortController class built-in e.g nodejs v14 and older.